relay/generate_docs.py

```diff
@@ -0,0 +1,16 @@
+import inspect
+import sys
+import workflow
+import pydoc
+
+p = pydoc.HTMLDoc()
+
+write_html = open("workflow.html", "w")
```

Comment:
use "with open" here and below : https://www.statology.org/with-open-python/

Reply:
Ah okay, that looks a lot cleaner - updated in latest commit